releases/SN30_Pro+/3.01/sn30pro_plus.metainfo.xml

```diff
@@ -25,19 +25,17 @@
     <release urgency="medium" version="3.01" date="2019-08-09" timestamp="1565309220">
       <checksum filename="Firmware_SN30_Pro+_V3.01.dat" target="content"/>
       <description>
-        <p>
-          This stable release fixes the following issues:
-        </p>
+        <p>This stable release fixes the following issues:</p>
         <ul>
-            <li>Fixed the latency issue when using the Switch with Joy-cons attached</li>
-            <li>Compatible with 8BitDo Ultimate Software</li> 
+          <li>Fixed the latency issue when using the Switch with Joy-cons attached</li>
+          <li>Compatible with 8BitDo Ultimate Software</li>
         </ul>
       </description>
     </release>
   </releases>
   <screenshots>
     <screenshot type="default">
-      <image type="source">https://raw.githubusercontent.com/hughsie/8bitdo-firmware/master/screenshots/sn30-pro.svg</image>
+      <image type="source">https://raw.githubusercontent.com/fwupd/8bitdo-firmware/master/screenshots/sn30-pro.svg</image>
```

Comment:
I don't think SVG is valid anymore. Can you use https://raw.githubusercontent.com/fwupd/8bitdo-firmware/master/screenshots/sn30-pro.png  pls

Reply:
Done.